sechub-wrapper-prepare/src/main/java/com/mercedesbenz/sechub/wrapper/prepare/prepare/PrepareWrapperPreparationService.java

```diff
@@ -0,0 +1,68 @@
+package com.mercedesbenz.sechub.wrapper.prepare.prepare;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.mercedesbenz.sechub.adapter.AdapterExecutionResult;
+import com.mercedesbenz.sechub.commons.model.*;
+import com.mercedesbenz.sechub.wrapper.prepare.cli.PrepareWrapperEnvironment;
+import com.mercedesbenz.sechub.wrapper.prepare.cli.PrepareWrapperResultStatus;
+
+@Service
+public class PrepareWrapperPreparationService {
+
+    private static final Logger LOG = LoggerFactory.getLogger(PrepareWrapperPreparationService.class);
+
+    @Autowired
+    PrepareWrapperEnvironment environment;
+
+    @Autowired
+    PrepareWrapperContextFactory factory;
+
+    @Autowired
+    PrepareWrapperResultStatus status;
+
+    public AdapterExecutionResult startPreparation() {
+        LOG.debug("Start preparation");
+        PrepareWrapperContext context = factory.create(environment);
+        List<SecHubRemoteDataConfiguration> remoteDataConfigurationList = getRemoteConfigurationsFromSecHubModel(context);
+        if (remoteDataConfigurationList.isEmpty()) {
+            // TODO: 27.03.24 laura is it ok that noe remote data was configured?
+            LOG.debug("No Remote configuration was found");
+            return new AdapterExecutionResult(status.getSTATUS_OK());
+        }
+
+        // TODO: 26.03.24 laura start GIT and SKOPEO Services with model
```

Comment:
Missing: The complete launch mechanism is missing.

I would expect here a something like

```java
@Autowired
List<PreparWrapperModule> modules = new ArrayList<>();

//...

for (PreparWrapperModule module: modules){
    var sechubConfiguration = context.getModel();
    if (module.isAbleToPrepare(sechubConfiguration){
        module.prepare(sechubConfiguration, remoteDataConfigurationList);
    }
}

```
The implementations could be done later, but currently
https://github.com/mercedes-benz/sechub/issues/2986 describes that the 
https://github.com/mercedes-benz/sechub/issues/3027 and https://github.com/mercedes-benz/sechub/issues/3028 should be done inside (but PR does only close 2986)

Remark:
Maybe it would be good to move the remoteDataConfigurationList calculation to a `PrepareWraperRemoteDataConfigurationService` which does this at startup and the modules just could have it injected - could keep the interface smaller...